packages/runtime/runtime-definitions/src/test/types/validateRuntimeDefinitionsPrevious.generated.ts

```diff
@@ -310,6 +310,7 @@ declare function get_current_InterfaceDeclaration_IContainerRuntimeBase():
 declare function use_old_InterfaceDeclaration_IContainerRuntimeBase(
     use: TypeOnly<old.IContainerRuntimeBase>): void;
 use_old_InterfaceDeclaration_IContainerRuntimeBase(
+    // @ts-expect-error compatibility expected to be broken
```

Comment:
For this and other breaks, please check Loop's codebase (Bohemia repo, maybe LoopSDK too) to see if they'll be affected by this rename or not.

Reply:
Gotcha. Sync up with @Leeviana Gray advanced. Should be good to go